app.js

```diff
@@ -36,18 +36,18 @@ const recoveryKeyJson = JSON.parse(process.env.ACCOUNT_RECOVERY_KEY);
 const keyStore = {
     async getKey(networkId, accountId) {
         if (accountId == creatorKeyJson.account_id) {
-            return KeyPair.fromString(creatorKeyJson.private_key);
```

Comment:
@chadoh we'll need to update secrets on Render based on this change before merging this PR. Alternatively we can support both `secret_key` and `private_key` to allow merging and deploying as is.